src/components/Table/Table.js

```diff
@@ -59,8 +62,19 @@ class Table extends Component {
   }
 
   render() {
+    let classes = classNames(
+      'sd-table',
+      'ui',
+      {'very basic': this.props.basic},
```

Comment:
Since the `very` class is used for `very padded` `very basic` and `very compact` I think we should expose it as a separate prop.  This way it is fully composable with the other uses.